### PR TITLE
Create a basic server

### PR DESCRIPTION
**Pull request by oluoluoxenfree**
hueniverse/hueniversity#1

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+var Hapi = require('hapi');
+var internals = require('../package.json');
```
Comment (idanwe) [inline lib/index.js]: internals should defined as object separately from the modules

File: lib/index.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+internals.response = { version: internals.version };
+
+var server = new Hapi.Server();
+server.connection({ port: 8000 });
+
+server.route({
+    method: 'GET',
+    path: '/version',
+    handler: function (request, reply) {
+        reply(internals.response)
```
Comment (idanwe) [inline lib/index.js]: add return

File: lib/index.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+          .type('application/json');
+    }
+});
+
+server.start(function () {
```
Comment (idanwe) [inline lib/index.js]: Add err handling

File: lib/index.js (hunk 1 above)
Comment (oluoluoxenfree) [inline lib/index.js]: How do you mean? 

The internals thing has confused me a bit but googling it hasn't brought up much.

Comment (idanwe) [inline lib/index.js]: Look at [eran's node.js coding style on the modules section] (https://gist.github.com/hueniverse/a06f6315ea736ed1b46d#module-classes)

File: lib/index.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
+var Hapi = require('hapi');
+var Hoek = require('hoek');
+
+var internals = {};
+internals.Json = require('../package.json');
```
Comment (hueniverse) [inline lib/index.js]: If you are going to keep the raw package file, don't use Uppercase variable name here (e.g. use `internals.json` instead).

File: lib/index.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+internals.response = { version: internals.Json.version };
+
+var server = new Hapi.Server();
+server.connection({ port: 8000 });
+
+server.route({
+    method: 'GET',
+    path: '/version',
+    handler: function (request, reply) {
+        return reply(internals.response).type('application/json');
```
Comment (hueniverse) [inline lib/index.js]: Missing empty line after function declaration.

**Pull request closed without merging**